dapp/src/pages/ProposalDetailsTab.vue

```diff
@@ -24,7 +24,7 @@
 			tab="Execute Proposal"
 			force-render
 		>
-			<ExecuteProposal />
+			<ExecuteProposal :updateStatus="getStatusOfProposal" />
```

Comment:
this is the code which wasn't working, we can remove this

Reply:
ok